server/gc/src/main/java/org/apache/accumulo/gc/replication/CloseWriteAheadLogReferences.java

```diff
@@ -104,28 +105,27 @@ public void run() {
     Span findWalsSpan = Trace.start("findReferencedWals");
     HashSet<String> closed = null;
     try {
-      sw.start();
+      startTime = System.nanoTime();
       closed = getClosedLogs(conn);
+      duration = Duration.ofNanos(System.nanoTime() - startTime);
     } finally {
-      sw.stop();
       findWalsSpan.stop();
     }
 
-    log.info("Found " + closed.size() + " WALs referenced in metadata in " + sw.toString());
-    sw.reset();
+    log.info("Found " + closed.size() + " WALs referenced in metadata in " + duration);
```

Comment:
Probbaily just a nit, but does changing the output format matter?  Duration will print out like: PT0.021850552S , Stopwatch is more "user" friendly with "333.2 ms"

Also, maybe unrelated by anything other than name, but there is a StopWatch class in org.apache.accumulo.core.util - it seems to be referenced only in org.apache.accumulo.server.client.BulkImporter.

Reply:
I tried really hard to preserve the format, without writing or copying code, and still supporting the various Guava versions, which had breaking changes in Stopwatch, and finally gave up. I don't think the format matters as much as the information... and I'm not sure the information even matters that much.